extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java

```diff
@@ -81,6 +86,11 @@ public void initialize() {
                 FullLight fullLight = getLight();
                 if (fullLight != null) {
                     updateProperty(Thing.PROPERTY_FIRMWARE_VERSION, fullLight.getSoftwareVersion());
+                    // Check for Osram PAR16 50 TW bulb
+                    String thingUID = HueBindingConstants.THING_TYPE_PAR16_50_TW.getId();
```

Comment:
Please rename it to thingTypeId, since the UID is defined to be the fully qualified name with colons in it and it is the id of the type, not of the thing.